tests/selenium/test_header.py

```diff
@@ -0,0 +1,134 @@
+import dash
+from dash.testing import wait
+
+from utils import get_props
+
+from dash_table import DataTable
+
+import pytest
+import time
+
+
+def get_app(props=dict()):
+    app = dash.Dash(__name__)
+
+    baseProps = get_props()
+    baseProps.update(
+        dict(columns=[dict(c, hideable="last") for c in baseProps["columns"]])
+    )
+
+    baseProps.update(props)
+
+    app.layout = DataTable(**baseProps)
+
+    return app
+
+
+def get_sigle_row_app(props=dict()):
+    app = dash.Dash(__name__)
+
+    baseProps = get_props()
+    baseProps.update(
+        dict(
+            columns=[dict(c, hideable=True, name=c["id"]) for c in baseProps["columns"]]
+        )
+    )
+
+    baseProps.update(props)
+
+    app.layout = DataTable(**baseProps)
+
+    return app
+
+
+@pytest.mark.parametrize("row", [0, 1, 2])
+def test_head001_renames_only_row(test, row):
+    test.start_server(get_app())
+
+    target = test.table("table")
+
+    title = [
+        target.column("rows").get_text(0),
+        target.column("rows").get_text(1),
+        target.column("rows").get_text(2),
+    ]
+
+    target.column("rows").edit(row)
+
+    alert = test.driver.switch_to.alert
+    time.sleep(2)
```

Comment:
That's a pretty long wait, isn't there some condition we can wait for instead?

Reply:
🔪 https://github.com/plotly/dash-table/pull/878/commits/4378ec00ae1d9d2520f9a2845c2d45479ae16037 - tests behave fine locally without it 🤞